app/styleguides/_default/components/footer.css

```diff
@@ -1,53 +1,54 @@
 .footer {
-  .logo-wrapper {
-    margin: 0 auto 15px auto;
-    width: 112px;
+  margin: 20px;
+  text-align: center;
+
+  .links {
+    display: inline;
+    width: 33px;
+    height: 14px;
+    font-family: Helvetica;
```

Comment:
Font should have a fallback in case the first one doesn't load. Also, remember to set them as variables at the top of the file

Reply:
🥇